.changeset/dry-poems-arrive.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/core-components': minor
```

Comment:
No, I don't consider it as a breaking change even some colors were changed using a different highlighting mode. I consider it more like a fix, because thee it's now more readable and it makes more sense not to show values as red. But I can mark it as a patch if you see it differently...